server/controllers/comment.js

```diff
@@ -0,0 +1,115 @@
+import models from "@models";
+import { httpResponse, serverError } from "@helpers/http";
+
+const { Comments } = models;
+
+/**
+ * @description Create a new Comment
+ *
+ * @param {object} req HTTP request object
+ * @param {object} res HTTP response object
+ * @returns {object}  Response message object
+ */
+export const createComment = async (req, res) => {
+  const articleId = req.article.id;
+  const { content } = req.body;
+  const { userId } = req.user;
```

Comment:
Correct me please, but I believe this userId property will not be coming from the client in this format, instead, it will be extracted from JWT sent by the client

Reply:
the userId is not coming from the user, its coming from the decoded token. If you check the middleware for verifying Token, you will see that this is returned: `req.user = decodedToken;`. Hence the `userId` is coming `req.user` which is coming from the decoded `token`

Reply:
you are both saying the samethings ? 🤷‍♂️